SDF-to-Coords

```diff
@@ -0,0 +1,36 @@
+import os
+
+import numpy as np
+
+def SDF_to_Coords(filename, num_atom):
+    directory = r'/mnt/d/SDFs/structures/'
+    for filename in os.listdir(directory):
+        if filename.endswith(".sdf"):
+            with open(filename) as f:
+                xyz = []
+                counter = 0
+                for line in f:
+                    counter += 1
+                    if counter < 4:
+                        continue
+                      
+                    elif counter > 4 and counter <= 4+num_atom: 
+                        line.replace(" ", "   ")
+                        line.replace("    ", "   ")
```

Comment:
The code, without these two lines, works. However, the spacing between elements in the xyz list output is inconsistent. Not to mention that the elemental symbols are not treated as separate list elements. With these lines, I'm trying to modify the file string so that the spacing between elements is consistent. If I can do this, the string split would then work perfectly. Do you have any suggestions?